docs/wmco-prerequisites.md

```diff
@@ -21,7 +21,7 @@ these errors, only use the appropriate version according to the cloud provider i
 | Cloud Provider | Supported Windows Server version                                                     |
 |----------------|--------------------------------------------------------------------------------------|
 | AWS            | Windows Server 2019, version 1809 Long-Term Servicing Channel (LTSC)                 |
-| Azure          | Windows Server 2019, version 1809 Long-Term Servicing Channel (LTSC)                 |
+| Azure          | - Windows Server 2019, version 1809 Long-Term Servicing Channel (LTSC)<br>- Windows Server 2022 Long-Term Servicing Channel (LTSC)                 |
```

Comment:
Format the MD table. See [this](https://github.com/openshift/windows-machine-config-operator/pull/1091/files#diff-7288c3766836fccd27562bcb1fc64927f66e877e9eadcdb8e030eed77a224738R21) change.